devices/flash-flexspi/fspi/fspi_rt105x.h

```diff
@@ -0,0 +1,112 @@
+/*
+ * Phoenix-RTOS
+ *
+ * Operating system loader
+ *
+ * FlexSPI Controller driver (i.MX RT105x)
+ *
+ * Copyright 2021-2022 Phoenix Systems
+ * Author: Gerard Swiderski
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+
+#ifndef _FSPI_RT105X_H_
+#define _FSPI_RT105X_H_
+
+
+/* clang-format off */
+
+enum { mcr0 = 0, mcr1, mcr2, ahbcr, inten, intr, lutkey, lutcr, ahbrxbuf0cr0, ahbrxbuf1cr0, ahbrxbuf2cr0, ahbrxbuf3cr0,
+	flsha1cr0 = 24, flsha2cr0, flshb1cr0, flshb2cr0, flsha1cr1, flsha2cr1, flshb1cr1, flshb2cr1, flsha1cr2, flsha2cr2,
+	flshb1cr2, flshb2cr2, flshcr4 = 37, ipcr0 = 40, ipcr1, ipcmd = 44, iprxfcr = 46, iptxfcr, dllacr, dllbcr, sts0 = 56,
+	sts1, sts2, ahbspndsts, iprxfsts, iptxfsts, rfdr32 = 64, tfdr32 = 96, lut64 = 128 };
+
+/* clang-format on */
+
+#define AHBRXBUF_CNT 4
+
+
+static inline void *flexspi_ahbAddr(int instance)
+{
+	return instance == flexspi_instance1 ? (void *)0x60000000 : NULL;
+}
+
+static inline void *flexspi_getBase(int instance)
+{
+	return instance == flexspi_instance1 ? (void *)0x402a8000 : NULL;
+}
```

Comment:
Defines might be simpler, e.g.:
#define FLEXSPI1_BASE (void *)0x402a8000
#define FLEXSPI2_BASE NULL